### PR TITLE
Add Muneem node framework

### PR DESCRIPTION
**Pull request by amitguptagwl**
Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [x] Please check configuration files
   + [x] `neph.yml`, for [neph](http://tbrand.github.io/neph) job processing
   + [x] `FRAMEWORKS.yml`, for **frameworks** list
   + [x] `travis.yml`, for CI
+ [x] A `Dockerfile` exists ?
+ [ ] All tests passes ?
~~~
export FRAMEWORK=<MY_FRAMEWORK>
shards install
shards build --release
bin/neph ${FRAMEWORK}
crystal spec
~~~

Addresses #748

### CODE REVIEW

File: FRAMEWORKS.yml [hunk 1]
```diff
@@ -36,6 +36,10 @@ node:
     github: jkyberneees/ana
     version: "2.3"
     language: "11.2"
+  muneem:
+    github: node-muneem/muneem/
+    version: "2.4.3"
```
Comment (waghanza) [inline FRAMEWORKS.yml]: We use X.Y format (the Z level is not very important to readers)

File: node/muneem/package.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "muneem": "~2.4.3"
```
Comment (waghanza) [inline node/muneem/package.json]: could use a strict dependency constraint, https://renovatebot.com/ will take care of update ?

**Comment by amitguptagwl** [comment]
@waghanza I'll update you as soon as I'm done with the proposed changes.

**Comment by waghanza** [comment]
@amitguptagwl I forget -> you have also to define a `neph` job in `neph.yaml`
~~~yml
muneem:
  commands:
    - docker build -t muneem .
  dir: node/muneem
~~~

**Comment by amitguptagwl** [comment]
grt! thanks. I'll do the same. Sorry for the delay. I'm working on anumargak (router of muneem framework). It is expected to be finished by tomorrow.

**Comment by waghanza** [comment]
btw, the actual results are on `docker` (**development**), I'm working on a **production** benchmark 
@see https://github.com/the-benchmarker/web-frameworks/pull/632#issuecomment-445024651

File: node/muneem/app.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+app.route({
+  url: "/user/:id",
+  to: function(req, res){
+      res.write(req.params.id);
+      res.end();
+  }
+})
+
+app.route({
```
Comment (waghanza) [inline node/muneem/app.js]: @amitguptagwl show use `POST` http method

@see https://travis-ci.com/the-benchmarker/web-frameworks/jobs/168545866#L514

Comment (amitguptagwl) [reply node/muneem/app.js]: done

**Comment by waghanza** [comment]
Thx @amitguptagwl , I'll update results (~3 hours)

**Comment by waghanza** [comment]
@amitguptagwl Results are updated https://github.com/the-benchmarker/web-frameworks#latency
however, little mistake, I was using `11.6` not `11.2` for **node** _benchmarks_

**Comment by amitguptagwl** [comment]
that's not a problem. Even I wanted to specify 11.6. But since all other repos have specified 11.2, I didn't change it. And many thanks for your quick action.

Some suggestions

1. Can we group framework language wise? It'll be easy to compare and find.
2. Some detail about the terms we are using. Eg in the second table if the throughput given in MB is good for small value or when it is a bigger value.

**Comment by amitguptagwl** [comment]
One more thing, are we updating the frameworks automatically? Because when I compare `restana: ^2.7.0` and `muneem:2.4.4" on my PC I found that restana is 20% slower. But on the README of this repo it is opposite.

**Comment by waghanza** [comment]
@amitguptagwl as you can see, for the `next` release, I use **production** release => https://github.com/the-benchmarker/web-frameworks/pull/632#issuecomment-445024651
the goal is to use a dataviz app to create beautiful graph ...

PS : The way of computing results in still on **development**, it differs of what you find on your workstation, but keep in track in above results, I'll try to add `node` base **frameworks**

**Comment by amitguptagwl** [comment]
Thanks @waghanza for the update

**Pull request merged**